packages/dashboard/src/components/cardGallery/components.js

```diff
@@ -17,114 +17,89 @@
 /**
  * External dependencies
  */
-import { forwardRef } from '@web-stories-wp/react';
 import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
-import { themeHelpers } from '@web-stories-wp/design-system';
+import { Button, BUTTON_TYPES } from '@web-stories-wp/design-system';
 
+const pictureCss = css`
+  picture {
+    display: block;
+    & > img {
+      display: block;
+      height: 100%;
+      width: 100%;
+      object-fit: fill;
+      border-radius: ${({ theme }) => theme.borders.radius.medium};
+      border: ${({ theme }) =>
+        `1px solid ${theme.colors.border.defaultNormal}`};
+    }
+  }
+`;
 export const GalleryContainer = styled.div`
   flex: 1;
   display: flex;
   justify-content: space-between;
   align-items: start;
 `;
 
-export const MiniCardButton = styled.button(
-  ({ theme }) => css`
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 100%;
-    width: 100%;
-    display: block;
+export const DisplayPage = styled.div(
+  ({ $isThreeRows }) => css`
     overflow: hidden;
-    cursor: pointer;
-    padding: 0;
-    background-color: transparent;
-    border: 1px solid ${theme.colors.border.defaultNormal};
-    border-radius: ${theme.borders.radius.small};
-    ${themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
+    width: ${$isThreeRows ? '45.2%' : '38.6%'};
+    margin-left: ${$isThreeRows ? '2.4%' : '2.1%'};
```

Comment:
We should be able to just do a media query for this now. Then we can remove the computing of it down in the component:
```suggestion
width: 38.6%;
margin-left: 2.1%;
@media screen and (min-width: 1600px) {
    width: 45.2%;
    margin-left: 2.4%;
 }
```

Reply:
you're totally right! Updated. :)